# BiorXiv paper journal club

# Single molecule multi-omics reveals context-dependent regulation of enhancers by DNA methylation

- Enhancers:
    - cis-regulatory elements 
    - control the establishment of cell identities during development
    - enhancer activation is tightly coupled with local DNA de-methylation

- SM multi-omics:
    - chromatin accessibility
    - transcription factor binding
    - as a function of the presence of methylation on the same DNA molecules

- test the impact of DNA methylation on activity across multiple cell lineages.

- de-methylation not necessary for enhancer activity in most cases
- Subset: cell-type-specific enhancers -> methylation "antagonises" TF-binding
    - genetic perturbation -> chromatin access + TF-binding determined by active de-methylation

- methylation mech.: 
    - indirect: recruitment of repressive complexes that compact chromatin
    - direct: inhibition of TF-binding
- enhancer: 
    - when activated = drop in methylation level -> sharpest differential methylation behaviour

- heterogeneous behaviour across cell population -> endogenous capacity to evluate effect of methylation level on activity within single cell population

- Single molecule footprinting:
    - measure-> co-occurence of methylation + chromatin access + nucleosome occupancy + TF-binding

-> most cases: methylation un-related to chromatin access or TF-binding
-> subset: enriched TF-binding for sites without methylation
    - genetic perturbation to validate
    - exogenous methylation to validate

- If methylation influences TF-binding at enhancers:
    - TF-binding at unmethylated sites > TF-binding at methylated sites

1. Footprint genome with M.CviPI -> target cytosine in CpG setting =/= methylated CpG
    - obstruction to target cytosine display different "signature" depending on the source of the obstructio

2. Measure continuous accessibility over 100bp -> distinguish nucleosome-bound vs open DNA
    - Proxy for TF-binding

- DNA methylation not reduced when chromatin accessible

    - focus on well covered + heterogeneous regions
    - subset enriched in enhancers, insulators, promoters
    - methylation heterogeneity correlated with accessibility
        - average methylation vs average accessibility?
            - simultaneous quantify the methylation + accessibility
    - 


- Focus on CpG subset with significantly lower accessibility when methylated -> 3.3% of sites
    - 2.6% explained through parental imprinting
    - majority away from promoters + histone marks indicating enhancer status
    - distinctive property = higher average accessibility (close to DNase peaks)

- Regulatory activity of this subset
    - determine consequence of perturbation
        - 2 genetic perturbations on methylation global level
            - remove
            - increase
        - Evaluate change in accessibility for subset and rest of sites
        - When removed methylation
            - only subset show substantial increase in accessibility
                - eg= 29% of sites have increase in CA
                    - what drop in methylation?
        - When increased methylation
            - subset only has decrease in CA
    - Could be correlated with potential target gene trx lvl
    
    - Upon differentiation subset of subset shown to significantly change in accessibility
        - could tied to genes relevant to considered transition

- Mech:
    - Assume binding factor for subset impacted by methylation in their binding affinity

    - Access ~ likely binding
   