# Journal Club: Sequence determinants of human gene regulatory elements

The temporal and spatial pattern of gene expression is encoded in the DNA sequence;
    this information is read and interpreted by transcription factors (TF), which recognize and bind specific short DNA sequence motifs. 

Major efforts have been undertaken to determine the DNA binding specificities of TFs in vitro and in mapping their binding positions in vivo. 

TFs regulate gene expression by binding to distal enhancer elements, and to promoters located close to the transcription start site (TSS). 

Both enhancers and promoters are characterized by: 
    RNA transcription 
    presence of open chromatin
    histone H3 lysine 27 acetylation 17 (H3K27ac). 

In addition, promoters and enhancers are marked by histone H3 lysine 4 (H3K4me1/3) trimethylation and monomethylation 18 , respectively. 

Although these features can be mapped genome-wide in a high-throughput manner, they are correlative in nature and do not establish that an element can act as an enhancer. (Ontology?)

To more directly measure enhancer activity, several investigators have developed massively parallel reporter assays (MPRAs) and used them to study the activity of yeast 19,20 , Drosophila 21,22 and human 23,24,25,26,27,28 gene-regulatory elements on a genome-wide scale. 

However, unbiased discovery of sequence-determinants of human gene expression by only analyzing genomic sequences is made difficult by the fact that the genome is repetitive and evolved to perform multiple functions in addition to transcription. 

Furthermore, the human genome is too short to even encode all combinations, orientations and spacings of the 1639 TFs in multiple independent sequence contexts.
    - Possible -> Real -> Necessary

Thus, despite the vast amount of information generated by the genome-scale experiments, most "Biologically Possible" sequence determinants that drive the activity of human enhancers and promoters, and the interactions between them remain unknown.

- putative enhancers are cloned to an exon, and the enhancer activity is then read using RNA-sequencing
Considered constructs:
1. synthetic TF motifs
    - consensus sequences flanked by two motif-depleted sequence contexts
    - only contained one type of consensus sequence, present once, twice and/or three times
    - Multiple consensus sequences were arranged in different spacings and orientations relative to each other
2. human genomic fragments
3. completely random synthetic DNA oligonucleotides
    - cloned upstream of the ORF in place of the minimal promoter and downstream of the ORF in the 3’ UTR
4. fourth library was designed to measure both enhancer and promoter activities of synthetic random 150 bp sequences.
cloned within the 3’UTR of the reporter gene (ORF) driven by 
- minimal δ1-crystallin gene (Sasaki) promoter or 
- CpG-free EF1α promoter

MPRA reporter libraries are transfected into human cell lines and total RNA is isolated after 24 hours of transfection followed by enrichment of reporter-specific RNA and Illumina library preparation, sequencing and data analysis. 
The active promoters are recovered by mapping *their* transcribed enhancers to the input DNA and taking the corresponding promoter pairs.
## Result

Set-up = [- Ultra-complex MPRA libraries with 100 times human genome coverage]

- Enhancers
    - from **known TF motifs**
        - Broad landscape and grammar of Enhancer activity  -> Weakly correlated with TF-expression and TF-binding
            - Few TFs display strong transcriptional activity in cells
            - Synergy, additivity and saturation of activity
    - **from random sequence**
        - Transcriptional enhancers can readily evolve de novo from random sequences
            - discover sequence features contributing to enhancer activity: random sequence set -> unbiased evaluation [unclear analysis]
                - moderate activity TF motif number increase = linear increase in activity
                - saturation when multiple high-activity TF motif (e.g. p53)
                - enrichment for intestinal TF (related to cell-line)
                - de novo = mostly individual TF or conventional heterodimers
                    - one strong composite motif = ETS-bZIP
            - ML model to characterise importance of all sequence features (Fig. 1)
                - Logistic regression with sequence motif to predict active vs inactive
                    - only handful necessary for optimal performance => L1 regularization
                        - pairwise combo didn't add much to prediction power
                        - Stongest contribution = tumerogenesis and colon development
                - CNN from sequence data
                    - better than logistic but with same motifs
                        - check method          
        - Transcriptional landscape of a cell is dominated by housekeeping TFs
            - random enhancer in HepG2
                - similar to GP5d
                - differences affecting lineage-determining TFs
                    - associated with reduced expression level for corresponding TFs
    - **from human genomic fragments**
        - Comparison of genomic enhancers and de novo evolved enhancers
            - Determine how sequence features combine to generate functional genomic enhancers
                - Assayed enhancer activity in GP5d and HepG2 cells using ~500 bp fragments derived from GP5d genome before and after methylation (?)
                - Determine the role of p53 in enhancer activity, we performed similar experiments also in p53 -/- GP5d cells (?)
            - association between accessible chromatin, TF binding and enhancer activity
                - STARR-seq peaks overlapped significantly with chromatin accessibility
                - ATAC-seq peaks could be predicted by a CNN from STARR-seq sequences
                    - trained using genomic or random STARR-seq sequences
                - overlap between ATAC-seq and STARR-seq was only partial
                    - six classes of elements
                        - STARR-seq+ : ATAC-seq- = closed chromatin enhancers
                        - silenced STARR-seq : regions = cryptic enhancers (?)
                        - ATAC-seq+ : with or without STARR-seq = promoters
                        - STARR-seq-/low : ATAC-seq+ : active histone marks = chromatin dependent enhancers
                        - STARR-seq- : ATAC-seq+ : CTCF+ = Structural chromatin elements
                        - STARR-seq+ : ATAC-seq+ = Classical enhancer
                    - 3 types of enhancers = classical, chromatin dependent/closed
                        - cryptic?
                        - appeared active based on the fact that inclusion of the  corresponding features improved prediction of differential gene expression between GP5d and HepG2 cells
                        - Analysis of ChIP-seq peaks and motifs
                            - classical and closed-chromatin enhancers = motifs similar to those found in active elements selected from random sequences
                            - Chromatin-dependent enhancers = motifs for FOXA + HNF4A
                    - STARR-seq peak motif analysis:
                        - enriched for relatively few motifs
                            - similar to known monomeric, dimeric and composite TF motifs determined using HT-SELEX 6 and CAP-SELEX
                        - motifs discovered from genomic and random enhancers were largely similar
                        - main difference was the discovery of the pioneer factor FOXA from genomic fragments
                            - points at the notion that enhancer activity beyond signature expression

- Promoters
    - **Promoters from random sequence (individual & paired?)**
        - A novel G-rich element that interacts with the TSS
            - evaluate the positioning of the different features relative to the TSS
                - template switching to recover 5'-end (TSS position) from within the promoters derived from random sequences
            - Alignment of the recovered sequences with respect to the TSS positions
                - classic Initiator motif at the TSS ~ high information content
                - AT-rich at -30 position of the TATA-box
                - not detect other TSS-proximal motifs that have previously been described
                - The +10 - +35 transcript side was characterized by a modest increase in G
                - mutual information analysis
                    - short-range interactions located 5'to the TSS
                        - represents enrichment of individual TF motifs
                    - 2 long-range interaction
                        - TATA-box and the TSS
                        - TSS and the G-rich downstream sequence
                        - loading of the RNA polymerase II either “heel first” (TFIID) or “toe first” (TFIIH) with respect to the TSS.
                - Motif mapping
                    - strongest positional signals
                        - TATA-box, Initiator and YY (YY1)
                    - YY1 motifs were mainly enriched on the transcript side
                    - many TF motifs preferentially enriched close to the TSS
                    - 5'end = decreasing trend from TSS
                    - 3'end = Even sharper decrease (no sigal after +20)
    - **Promoters from random sequence (individual & paired?) and genomic**
        - Prediction of cellular transcriptional activity using the sequence features
            - Can transcription be predicted from the evolved promoter sequences
                - used the sequences to train a CNN model for predicting the TSS positions genome-wide
                    - score wild-type and mutant forms of the TERT promoter
                        - correctly predicted that known cancer-associated mutations increase the activity of this promoter
                    - predict the positions of active TSSs in GP5d cells
                        - random sequences were more predictive than the genomic sequences themselves
                        - trained on STARR-seq data = stronger position-specific signal
                        - trained on EPD data = information present at a relatively short region around the TSS
- Paired promoter-enhancer
    - **Paired enhancer-promoter random sequence**
        - Sequence features of promoters and enhancers evolved from random sequences
        - Local enhancer-promoter interactions are additive and non-specific in nature


## Discussion

Learning the rules by which DNA sequence determines where and when genes are expressed has proven surprisingly hard, 
    despite the availability of full genome sequences of several mammals 48 , extensive maps of genomic features 9,10,15,18 , and genome-scale data about TF protein expression levels and TF DNA binding in vitro 6,8,49 . 
    Direct comparison of activities between TFs has remained difficult, and therefore we generally lack parameters describing the relative strength of the different sequence features and their interactions that are required to predict their aggregate activity. 
To address this, we have here defined sequence determinants of human regulatory element activity in an unbiased manner, using a molecular evolution approach where genomic, designed and random sequences are identified that display promoter or enhancer activity.

We found that the cellular gene regulatory system is relatively complex, consisting of several distinct kinds of elements. 
However, the motif grammar within the elements is relatively loose. 
    In the evolved sequences, precise TF arrangements such as those found in the interferon enhanceosome 50,51 are rare, with most elements consisting of TFs acting together largely in an additive manner 30,31,32,52,53,54,55,56 . 
    Cells are very similar to each other, and the topology of the gene regulatory network is hierarchical, with few TFs displaying very strong transactivation activity. 
Our findings contrast with the known tissue-specificity of many putative enhancer elements in vivo 57,58 . => **Low specificity**
Interestingly, the level of conservation of many endogenous promoters and enhancers appears to be higher 59 than the elements selected in our assay (Fig. S10b). => **Low conservation**
    The simplest explanation for these two facts is that enhancers in vivo [evolve to be specific], and that due to the similarity between cells, specificity is more difficult to achieve than activity. 
    Specificity is also required to silence strongly active elements in cell types where a protein is not needed, due to the significant fitness cost of protein expression 60 .
        - Not evaluated/selected for by this assay
    - Limit of such generic (pan-genome assays) => detect the dry chemical invariant on top of which the organic processes unfold

The original functional definition described enhancers as genetic elements that can activate a promoter from a distance, irrespective of their orientation relative to the TSS 1 . 
    - We find here that in addition to these elements, two other types of enhancing gene regulatory elements exist: -> unclear to what degree these new element extend the coverge of endogenous reg. elements
        - chromatin-dependent enhancers:
            -  The chromatin-dependent enhancers are characterized by 
                - forkhead motifs, and 
                - binding of Mediator and p300 protein, and 
                - strong signal for H3K27 acetylation. 
                - Unlike classical enhancers, chromatin-dependent enhancers do not transactivate a heterologous promoter strongly, most likely due to lack of binding of TFs with strong transactivator domains. 
                - Their presence is, however, strongly predictive of tissue-specific gene expression, suggesting that they act to increase gene expression via chromatin modification or structural changes in higher-order chromatin. 
                - Several chromatin-dependent enhancers also combine with a single classical enhancer to form super-enhancers (see Fig. 2b), indicating that these elements may be required for driving high levels of gene-expression from distal promoters.
        - closed chromatin enhancers: Closed chromatin enhancers, in turn, are 
            - located in regions that show little or no signal for DNase I hypersensitivity or ATAC-seq. 
            - They are not silenced by CpG methylation. 
            - These elements appear to consist of only a single TF (e.g. p53; see also ref. 61 ) or a set of closely bound TFs that fit between or associate directly with well-ordered nucleosomes. 
    - The prevalence of both the closed chromatin enhancers and chromatin-dependent enhancers suggests that they may contribute significantly to regional control of gene expression 56,62 .

By using machine learning approaches, we show here that transcriptional activity in human cells can be predicted from sequence features. 

Interestingly, we found that the promoters enriched from completely random synthetic sequences in a single experimental step are even more predictive of transcriptional activity than the genomic sequences themselves. 
- By analysis of de novo evolved promoters, we discovered a novel G-rich element that interacts with the TSS, potentially positioning RNA polymerase II to the TSS independently of the TATA box. 

- Overall, TF activities could be classified into three groups: 
    - TSS position determining activity (e.g. TATA-box, YY), 
    - short range promoting activity (e.g. NRF1), and 
    - enhancing activity (many TFs). 
We did not detect a separate class of distal enhancing activity, suggesting that activities that would allow an enhancer to selectively act at a very long range is likely to be associated with chromatin-dependent enhancers and not classical enhancers 67,68,69,70,71 . 
The three classes of activities detected are not mutually exclusive, suggesting that TFs act at multiple levels and/or scales to regulate transcription (Fig. 5d). 
    - For example, YY1 acts both as an enhancing TF and as a TSS-determining one, and that FOXA motifs are present at both chromatin-dependent and classical enhancer elements. 
    Our results thus indicate that gene regulatory elements are not the atomic units of gene expression, and 
        - that TF motifs should ultimately be the basis of analysis and prediction of genomic gene regulatory activity.

Our random promoter-enhancer design allowed unbiased discovery of features that facilitate interactions between classical enhancers and promoters at a relatively short range. 
No specific pair of motifs controlling such interactions was found. 
    - This, together with the fact that no specific TF that only acts from an enhancer was found is consistent with a generic and indirect mechanism of action, where the activities of individual TFs bound to an enhancer are aggregated, and their total activity then activates the promoter. 
    - Molecularly, these results are consistent with mediation of the effect by the least specific type of biochemical interaction, steric hindrance. 
        - The simplest mechanism for enhancer action would involve opening of higher order and local chromatin in such a way that the steric hindrance that prevents large macromolecular complexes such as mediator or RNA polymerase II from loading to DNA is decreased (Fig. 5d; see ref. 72 ). 
        
In summary, we show here that direct experimentation to interrogate transcriptional activities of sequences that are on aggregate >100 times longer than the human genome can be used to determine mechanisms of action of, and interaction between, gene regulatory elements. 

The experiments revealed unexpected simplicity of gene regulatory logic. The discovery of the relative simplicity of the interactions, together with the ability to measure transcriptional activities of all TFs in a cell represents a significant step toward achieving the ultimate aim of regulatory genomics – predicting gene expression from sequence.

# Resutl slide flow

- What Data
- What Question
    - 1st sentence of each section
- What analysis/Method
- What main outcome
