# Paper and Review notes

## The relationship between genome structure and function
- Coordinated cell-activity achieved through coordinated activation of gene-set transcription (signature) *encoded* in genome (better to say in large part determined by genome) and *decoded* by protein factors binding targeted sequences (better to say realised by protein aggregating at targeted regions enabling the appropriate transcription). Genome determination carries some "history" in the form of epigenomic patterning promoting particular gene set activation.
- Investigating this coordination has highlighted three important kinds of regions: promoters, enhancers and boundary elements representing collectively 5-15% of the genome

- Relation to chromatin 3D configuration= causative or reflective of regulatory activity? -> arbitrary dichotomy between reg. elements and chromatin 3D, both reflect the regulatory necessity for cell activity with neither having an exclusive right to realise it. => unilateral definition of gene regulation
    - causative = architectural principle?
    - reflective = how regulatory activity drives emergence of 3D structure? => rather is 3D patterning of regulatory activity obeying any logic?-> 3D chromatin structure being one material expression of regulatory necessity, can we infer any logic for the latter from the chromatin's 3D configuration?
        - co-determination rather than causation => effective outcome is coordinated cell activity, neither of which can claim to fully explain, escapes the generic additive paradigm

- Defining the reg. element (taxonomy) => Aristotle: Finding closest genus and most distinctive property
    - **Promoter**: Core= minimal seq able to initiate *basal gene expression*
        - TSS -> 35-40 bp serving as landing platform for pre-initiation complex
            - Trx observed at these sites result from the "integration" of the various "regulatory stimuli acting around it"
            - Bi-directional Trx but elongation favoured downstream towards gene body, opposite direction produce short and short-lived RNA products
            - E.g.= TATA-box, Initiator elements, downstream promoter element, downstream core element and the upstream and downstream TFIIB recognition element
            - Just G-C rich can suffice to associate with effective Trx.
        - TSS proximal elements = within the same nucleosome-​free region (40–250 bp upstream of TSS)
    - **Enhancers**: short DNA regions "activating target regions independantly" of distance or orientation + "rich" in binding sites for various TF => contribute towards the determination of the outcome from signalling integration at promoters
        - "Strength" of enhancers determined by TF-affinity profile
            - TF affinity + variety + typology => found exceptional enhancers with distinct properties, called super-enhancers
        - More numerous than genes
        - Operate as clusters, the composition of which modulates the *signalling* cocktail exposed to the target TSS. 
    - **Boundary elements**: DNA regions able to modulate 1) spread of Heterochromatin 2) enhancer-promoter coupling
        - regions with assymetric interaction profile => shape the chromatin context of TSS

- 3D phenomenological description of chromatin structure:
    - Assortative aggregation based on epigenomic content (compartment) segmenting the chromosome into active and inactive "blocks" => can be predicted from epi-content
    - Local (<3Mb) interaction asymmetry sometimes driven by loop-extrusion forming prefferentially self-interacting modules.
        - Regulatory ores -> condensate of factor binding
            - Composition : variety + affinity + typology
            - Position/Context

- Most pressing question for Str-Fn coupling at the scale of TADs = E-P coupling determination
    - Loop extrusion -> magic bullet to rationalise association between chromatin interaction pattern and reg-element coordinated activity    




# Ieva group
RSG Norway is the ISCB’s official regional student group in Norway. 
We are a non-profit organisation aiming to create an inclusive space fostering interactions and networking among computational biology practitioners of all levels.

This meet-&-greet will introduce our main activities and goals and open a discussion about how we as an organisation can best address the new challenges and needs the current pandemic raises 



- 1ere moitie sur le proces et ce que cette affaire dit de l'etat actuel du traitement managerial/judiciaire des travailleurs
- Entracte de Vamos
- 2eme moitie on couvre le vecu de sefran
    - logiciel libre comme matrice de l'etat d'esprit de sefran
    - Defense de ces principes au sein de pole emplois?
    - (Biscuit) Application de ces principes avec electriciens sans frontieres?

#======================================================================================

This job failed during processing. Please check error log or contact us with the job URL: https://unibind.uio.no/enrichment/UniBind_oneSetNoBg_uq106yw2_20220128/


Dear David Mas-Ponte,

My name is Vipin Kumar, I am a postdoctoral fellow working at the Mathelier Lab in Oslo. Last week I had the opportunity to exchange with Dr. Fran Supek who mentioned your recent work on omikli using the hyperClust package you developed. 

As part of my research I was hoping to use an appropriate null model for cancer variants and it appears that this is an important element of the methodology that allowed the robust detection of omikli .

I was hoping if we could maybe discuss ways I could solicit your method for producing this "null" distribution of cancer variants for my particular application.

Thanking you in advance,
Best regards,
Vipin.


https://www-pnas-org.ezproxy.uio.no/doi/abs/10.1073/pnas.1518552112
https://www-sciencedirect-com.ezproxy.uio.no/science/article/pii/S2211124716305307


hg38_permissive_UniBind_LOLA.RDS
hg38_permissive_UniBind_LOLA_universe.RDS
hg38_robust_UniBind_LOLA.RDS
hg38_robust_UniBind_LOLA_universe.RDS
mm10_permissive_UniBind_LOLA.RDS
mm10_permissive_UniBind_LOLA_universe.RDS
mm10_robust_UniBind_LOLA.RDS
mm10_robust_UniBind_LOLA_universe.RDS

bin/zenodo_fetch 4161027 -f hg38_permissive_UniBind_LOLA.RDS -f hg38_permissive_UniBind_LOLA_universe.RDS -f hg38_robust_UniBind_LOLA.RDS -f hg38_robust_UniBind_LOLA_universe.RDS -f mm10_permissive_UniBind_LOLA.RDS -f mm10_permissive_UniBind_LOLA_universe.RDS -f mm10_robust_UniBind_LOLA.RDS -f mm10_robust_UniBind_LOLA_universe.RDS

